orchagent/macsecorch.cpp

```diff
@@ -218,6 +220,68 @@ static void lexical_convert(const std::string &buffer, MACsecAuthKey &auth_key)
     }
 }
 
+class MACsecSCI
+{
+public:
+    operator sai_uint64_t () const
+    {
+        SWSS_LOG_ENTER();
+
+        return m_sci;
+    }
+
+    std::string str() const
+    {
+        SWSS_LOG_ENTER();
+
+        return boost::algorithm::to_lower_copy(swss::binary_to_hex(&m_sci, sizeof(m_sci)));
```

Comment:
Is using `boost::algorithm::to_lower` not ok?

Reply:
to_lower is in-place conversion.